README.md

```diff
@@ -15,7 +15,8 @@ export-interfaces exposes the 'Export Manager' interface with common functions t
 ```typescript
 export interface IExportManager {
   createExportTask: (data: CreateExportTaskRequest<TaskParameters>) => Promise<CreateExportTaskResponse>;
-  getEstimations: (params?: GetEstimationsRequest) => Promise<GetEstimationsResponse>;
+  getEstimations: (catalogRecordID: string, ROI: FeatureCollection, additional?: Record<string, unknown>) => Promise<GetEstimationsResponse>;
```

Comment:
shouldn't the "additional" be an array?

Reply:
No, it’s an kvp properties
But anyway Alex claims it’s should be removed
Should talk to him